client-v2/src/keyboard-shortcuts/init.ts

```diff
@@ -0,0 +1,29 @@
+import { Store } from 'types/Store';
+import {
+  editorCloseAllOverviews,
+  editorOpenAllOverviews,
+  selectIsClipboardOpen,
+  editorCloseClipboard,
+  editorOpenClipboard
+} from 'bundles/frontsUIBundle';
+import { trap } from './trap';
+
+const createTrapMap = (store: Store) => ({
+  'command+j': () => {
```

Comment:
how does this map on non macs?

Reply:
Good point. It doesn't, although this key binding is _secret_. It's there to test the implementation, I'm not even sure if that's a good binding even on mac. I think `command` is synonymous with `meta` which I _think_ is the windows key on windows for example.